test/ex_doc/autolink_test.exs

```diff
@@ -449,11 +450,17 @@ defmodule ExDoc.AutolinkTest do
     assert warn(~m"[bad](`String.upcase/9`)", extras: []) =~
              "documentation references \"String.upcase/9\" but it is undefined or private"
 
-    assert warn(~m"[custom text](`Elixir.Unknown`)", []) =~
-             "documentation references module \"Elixir.Unknown\" but it is undefined\n"
+    assert_and_warn(
+      ~m"[custom text](`Elixir.Unknown`)",
+      ["custom text"],
+      "documentation references module \"Elixir.Unknown\" but it is undefined\n"
+    )
 
-    assert warn(~m"[Unknown](`Unknown`)") =~
-             "documentation references module \"Unknown\" but it is undefined"
+    assert_and_warn(
+      ~m"[It is Unknown](`Unknown`)",
+      ["It is Unknown"],
+      "documentation references module \"Unknown\" but it is undefined"
+    )
 
```

Comment:
which test is testing the changed behaviour?

Reply:
the ones that call assert_and_warn

Reply:
I might be missing something but it looked like you changed `assert warn(...)` to `assert_and_warn(...)` with the same inputs so effectively we are effectively not adding additional test coverage?

Reply:
it is doing an assert between the first and the second argument which checks for the link being stripped

Reply:
oh, I see it now, previously we weren't checking what's the generated output. thanks!